card-data/01-character-focus.yaml

```
@@ -30,12 +30,14 @@
     desc: >
       //"Emory! Fall back! This isn't our fight!" -- Tana
 
-      Not every risk has to be taken, nor is every reward worthwhile.
-      Sometimes, you do things it would have been better not to do.
+      Courage can twist into cowardice or cockiness.
```

```suggestion
      Courage can twist into cowardice or recklessness.
```
I think this wording might be a bit more direct to grasp as a prompt.